CMakeLists.txt

```diff
@@ -98,32 +75,33 @@ option(ENABLE_DOCS
       "If docs should be generated or no" OFF)
 option(BUILD_RUN "" ON)
 
-# set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
+# Include pathstests
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-set(BASEPATH "${CMAKE_CURRENT_LIST_DIR}")
 set(INCLUDE_PATHS
   ${BASEPATH}
   ${math_SOURCE_DIR}
   ${math_SOURCE_DIR}/lib/boost_1.78.0
   ${math_SOURCE_DIR}/lib/eigen_3.4.0
   ${math_SOURCE_DIR}/lib/sundials_6.1.1/include
-  ${TBB_ROOT}/include
+  ${math_SOURCE_DIR}/lib/tbb_2020.3/include
   ${CMAKE_CURRENT_BINARY_DIR}
-  ${protobuf_SOURCE_DIR}/src
+  ${Protobuf_INCLUDE_DIRS}
 )
 
+# Link paths
 set(LINK_LIBRARIES
 	pthread
 	protobuf::libprotobuf
-	TBB::tbb
+  tbb
   OpenMP::OpenMP_CXX
```

Comment:
Not sure what's going on with the indentation here?

Reply:
Guess it's just a problem of tab/space 🤔